.gitignore

```diff
@@ -52,3 +51,6 @@ $RECYCLE.BIN/
 # Mac desktop service store files
 .DS_Store
 bin
+
+# anvil-csharp-core Specific
+Logging/anvil-csharp-logging.deps.json
```

Comment:
Just curious, where does this file come from, and why ignore it?

Reply:
Rider won't auto generate but VS for Mac will  (probably Windows too). I believe it's used to track NuGet dependencies which anvil doesn't support because Unity doesn't have native support.

For whatever reason this hasn't come up with the root level csproj file.